lib/renderer/init.js

```diff
@@ -27,23 +27,11 @@ v8Util.setHiddenValue(global, 'ipc', new events.EventEmitter())
 // Use electron module after everything is ready.
 const { ipcRenderer } = require('electron')
 
-const {
-  warnAboutNodeWithRemoteContent,
-  warnAboutDisabledWebSecurity,
-  warnAboutInsecureContentAllowed,
-  warnAboutExperimentalFeatures,
-  warnAboutEnableBlinkFeatures,
-  warnAboutInsecureResources,
-  warnAboutInsecureCSP,
-  warnAboutAllowedPopups,
-  shouldLogSecurityWarnings
-} = require('@electron/internal/renderer/security-warnings')
-
 require('@electron/internal/renderer/web-frame-init')()
 
 // Process command line arguments.
-let nodeIntegration = 'false'
```

Comment:
This change seems unrelated and concerns me slightly, I agree this variable should probable be a boolean rather than a string representation of a boolean but I think this change needs more scrutiny and doesn't belong in this PR.  Can you extract this change into a different PR.

Reply:
will do that

Reply:
@MarshallOfSound done